roles/load-ycsb/tasks/main.yml

```diff
@@ -35,9 +35,9 @@
               - name: es_port
                 value: "{{elasticsearch.port | default()}}"
               - name: es_index
-                value: "{{ es_index }}"
+                value: "{{ elasticsearch.es_index | default('ripsaw-ycsb') }}"
               - name: user
-                value: "{{test_user | default()}}"
+                value: "{{test_user | default('ripsaw') }}"
```

Comment:
"{{test_user | default('ripsaw')}}" or "{{ test_user | default('ripsaw') }}" for consistency